### PR TITLE
Adds the timerManager class 

### LINKED ISSUE #23: Add a TimerManager that would handle timers and resets them automatically if necessary
Here's a quick and dirty implementation example to get you started :

```java
public final class TimerManager {
	
	private static HashMap<String, Thread> timers;
	
	private TimerManager(){}
	
	/**
	 * @param timerName Name of the timer
	 * @param delay Delay (in milliseconds) before doing the action
	 * @param action Action to do after the delay was finished, without being called again, which resets the delay
	 */
	public static void manage(final String timerName, final int delay, Runnable action){
		
		if(timers == null)
			timers = new HashMap<>();
		
		stopTimer(timerName);
		
		Thread timerThread = new Thread(() -> {
			
			try{
				
				Thread.currentThread().sleep(delay);
				
				action.run();
				
			}
			catch(InterruptedException e){}
			finally{
				timers.remove(timerName);
			}
			
		});
		
		timers.put(timerName, timerThread);
		
		timerThread.start();
		
	}
	
	public static void stopTimer(String timerName){
		
		if(timers.containsKey(timerName))
			timers.get(timerName).interrupt();
		
	}
	
}
```

### PR DESCRIPTION
**Pull request by StephanoMehawej**
This pull request add's the TimerManager class that allows commands to start a timer and at the end of it, it will perform a task. closes #23

### CODE REVIEW

File: src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	private static HashMap<String,Timer> timers;
+
+	private TimerManager(){}
+
+	/**
+	 * @param timerName Name of the timer
+	 * @param delay Delay (in milliseconds) before doing the action
+	 * @param action Action to do after the delay was finished, without being called again, which resets the delay
+	 */
+	public static void manage(final String timerName, final int delay, TimerTask action){
```
Comment (V-ed) [inline src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java]:
```suggestion
	public static void manage(final String timerName, final int delay, Runnable action){
```

I would make that change so using TimerManager is as simple as possible, and the `TimerTask` object would be created inside here automatically.

Comment (V-ed) [inline src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java]: Can we change the method name to `schedule`, therefore matching the way `java.util.Timer` already does it? It makes more sense anyway.

Comment (StephanoMehawej) [reply src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java]: I am not sure that would actually work but i'll check it out

Comment (V-ed) [reply src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java]: Just doing that change wouldn't work indeed, but what I wanted to emphasize is the parameters : the method should have a runnable, and inside, it creates a new `TimerTask` ("automatically") and runs `action.run()`.

I suggest to change the `TimerTask` parameter to a `Runnable`, obviously the inner-working logic will need to change too ;)

Comment (StephanoMehawej) [reply src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java]: Yeah i realised that after asking

**Pull request merged**